src/dotnet/commands/dotnet-new/CSharp_Web/$projectName$.csproj

```diff
@@ -1,14 +1,21 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Let's talk about this one in person tomorrow if possible. There seem to be 3 phases in LZMA archive creation/extraction at first run.

1. The package references from a template (this one) are used to compute a hash that's used as a blob name to fetch, allowing the creation of the archive to be skipped if it already exists
2. The template (via an invocation of new) is used to catalog the actual references to use, these are restored into a directory, packed up & uploaded to a blob (iff phase 1 couldn't find a matching archive)
3. At first run, a set of templates from invocations of new (I think) is restored to the user cache of packages

I'm not sure why phase 1 didn't use an invocation of new in the first place, but this now poses a synchronization issue if it doesn't.